rtp/src/main/kotlin/org/jitsi/rtp/rtp/header_extensions/Av1DependencyDescriptorHeaderExtension.kt

```diff
@@ -623,7 +627,7 @@ class Av1DependencyDescriptorReader(
          */
         templateCnt = 0
         templateInfo.clear()
-        decodeTargetInfo.clear()
+        decodeTargetLayers.clear()
```

Comment:
Does `decodeTargetProtectedBy` also need to be cleared?

Reply:
Ah, you're right, yes, thank you!